ocs_ci/ocs/resources/pvc.py

```diff
@@ -165,3 +165,38 @@ def get_all_pvcs(namespace=None):
     )
     out = ocp_pvc_obj.get()
     return out
+
+
+def wait_for_pvc_count_change(
+    previous_num, namespace, change_type='increase', min_difference=1,
+    timeout=20, interval=2
+):
+    """
+    Wait for a change in total count of PVC
+
+    Args:
+        previous_num (int): Previous number of PVCs to compare
+        namespace (str): Name of the namespace
+        change_type (str): Type of change to check. Accepted values are
+            'increase' and 'decrease'
+        min_difference (int): Minimum required difference in PVC count
+        timeout (int): Maximum wait time in seconds
+        interval (int): Time in seconds to wait between consecutive checks
+
+    Returns:
+        bool: True if difference in count is greater than or equal to
+            'min_difference'. False in case of timeout.
+    """
+    try:
+        for sample in TimeoutSampler(
+            timeout, interval, get_all_pvcs, namespace
+        ):
+            current_num = len(sample['items'])
```

Comment:
get_all_pods() and get_all_pvcs should return the same typle of output right?
Here it is sample['items'] and in get_all_pods you just checking the len(sample).

the main reason for this comment is to combine those 2 methods to one and pass the function to execute (get_all_pods, get_all_pvcs)

Reply:
The return type is different for both.
get_all_pods() returns list: List of Pod objects
get_all_pvcs() returns dict: Dict of all pvc in namespaces

Okay,
Will combine wait_for_pvc_count_change() and wait_for_pod_count_change() and add in helpers.py

Reply:
Done